plugin/src/main/java/com/craftmend/openaudiomc/generic/redis/packets/adapter/RedisTypeAdapter.java

```diff
@@ -17,7 +17,7 @@ public JsonElement serialize(OARedisPacket src, Type typeOfSrc, JsonSerializatio
         JsonObject result = new JsonObject();
 
         result.add("type", new JsonPrimitive(src.getClass().getName()));
-        result.add("payload", context.serialize(src, src.getClass()));
+//        result.add("payload", context.serialize(src, src.getClass()));
```

Comment:
i temporarily commented this because it caused an infinite loop, calling serialize() recursively on `src`